client/src/elements/schema-editor/schema-editor-table.js

```diff
@@ -81,6 +81,17 @@ function hasNonEmptyProperty(obj, excludedProperties = []) {
     });
 }
 
+function isNumeric(cell) {
+  if (!cell) {
+    return false;
+  }
+  cell = cell.trim(); // remove whitespaces
+  if (cell.match(/^[+-]?\d+(\.\d+)?$/) === null) {
```

Comment:
check the regex whether it is a number?
I would make a comment what the regex does. Then you can see at a glance what the regex is doing. Nice to have :wink: